system/ee/legacy/core/Input.php

```diff
@@ -259,6 +259,18 @@ protected function _set_cookie(array $data)
 		/*
 		/* -------------------------------------------*/
 
+		if (PHP_VERSION_ID < 70300) {
+			// Older versions of PHP do not support an array as the 3rd parameter,
+			// thus the SameSite setting must be hacked in with the path option.
+			return setcookie($data['prefix'].$data['name'], $data['value'],
```

Comment:
For reference https://stackoverflow.com/questions/39750906/php-setcookie-samesite-strict